src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -187,7 +191,9 @@ const Footer = BaseFooter.extend({
         options: {
           name: 'cancel-authenticator-challenge',
           label: loc('loopback.polling.cancel.link', 'login'),
-          actionPath: CANCEL_POLLING_ACTION,
+          clickHandler: () => {
```

Comment:
my understanding on why we don't have USER_CANCELLED at authentication step:
1. if user clicks Verify with something else, there will be no api call and user just fallback to factor selection page based on previous remediaton `select-authenticator-authenticate`
2. if user clicks cancel in OV, it's OV that will send the request to server so SIW can't do anything until they receive next polling response
If so, pls add a comment here to state the diff so that ppl know this is intentional